MyApplication/app/src/main/java/com/example/ivanbui/twittersearch/MyAdapter.java

```diff
@@ -18,52 +18,55 @@
  * Created by ivanbui on 2017-05-04.
  */
 
-public class MyAdapter extends RecyclerView.Adapter<MyAdapter.MyViewHolder>{
+public class MyAdapter extends RecyclerView.Adapter<MyAdapter.TweetHolder>{
     private LayoutInflater inflater;
-    List<Status> data = Collections.emptyList();
     List<MyTweet> statusList;
 
     public MyAdapter(List<MyTweet> statusList){
         this.statusList = statusList;
     }
 
     @Override
-    public MyViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-
+    public TweetHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View view = inflater.from(parent.getContext()).inflate(R.layout.list, parent, false);
-        return new MyViewHolder(view);
+        return new TweetHolder(view);
     }
 
     @Override
-    public void onBindViewHolder(MyViewHolder holder, int position) {
+    public void onBindViewHolder(TweetHolder holder, int position) {
         MyTweet current = statusList.get(position);
-        holder.text.setText(current.text);
-        holder.name.setText(current.name);
-        holder.screenName.setText(current.screenName);
-
-        String url = current.imgUrl;
-        Context context = holder.profilePic.getContext();
-        Picasso.with(context).load(url).into(holder.profilePic);
+        holder.bind(current);
     }
 
     @Override
     public int getItemCount() {
         return statusList.size();
     }
 
-    class MyViewHolder extends RecyclerView.ViewHolder{
+    class TweetHolder extends RecyclerView.ViewHolder{
+
         TextView text;
```

Comment:
Better name for textView other than text, name, screenName. Ex: The string will probably be name so you dont' want a line like name.setText(name) it would look better as nameTextView.setText(name)